src/MAUI/Maui.Samples/Samples/Layers/AddVectorTiledLayerFromCustomStyle/AddVectorTiledLayerFromCustomStyle.xaml

```diff
@@ -0,0 +1,15 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<ContentPage x:Class="ArcGIS.Samples.AddVectorTiledLayerFromCustomStyle.AddVectorTiledLayerFromCustomStyle"
+             xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:esri="clr-namespace:Esri.ArcGISRuntime.Maui;assembly=Esri.ArcGISRuntime.Maui">
+    <Grid Style="{DynamicResource EsriSampleContainer}">
+        <esri:MapView x:Name="MyMapView" Style="{DynamicResource EsriSampleGeoView}" />
+        <Border Style="{DynamicResource EsriSampleControlPanel}">
```

Comment:
Suggest replacing the `Border` element with the following:

```
<Border MaximumWidthRequest="300" Style="{DynamicResource EsriSampleControlPanel}">
            <VerticalStackLayout Padding="5" Spacing="5">
                <Label x:Name="ChosenStyle" VerticalOptions="Center" />
                <Button Clicked="ButtonClick" Text="Change style" />
            </VerticalStackLayout>
        </Border>
```

I think the vertical stack and maximum width looks like a cleaner UI - your call.

Reply:
This looks nice!